pagopa/io-app
# chore(IT Wallet): [SIW-963] Add wallet cards onboarding screen

> [!WARNING]
> This PR depends on https://github.com/pagopa/io-app/pull/5645

## Short description
This PR implements the cards onboarding screen in the new wallet section

## List of changes proposed in this pull request
- Updated `@pagopa/io-app-design-system` tom version 1.33.0
- Added `WalletCardOnboardingScreen` component
- Added navigation components
- Added navigation to the wallet screen after CGN activation success

## How to test
With the` io-dev-api-server`, enable the **New wallet section** FF from the profile section. Navigate to the wallet screen and tap on the CTA. Check that the screen renders and works as expected.

## Preview

https://github.com/pagopa/io-app/assets/6160324/542b71ed-043d-4357-8492-8b85003b1ea8
